packages/app/src/components/feature/screens/travel/TravelPauseButton/index.tsx

```
@@ -1,27 +1,32 @@
 import useTravelStateStore from "@store/travel";
+import * as Location from "expo-location";
 import { useLocalSearchParams } from "expo-router";
 
 const TravelPauseButton = () => {
   const { courseId } = useLocalSearchParams();
   const socketManager = SocketManager.getInstance();
   const { connectedURL } = useTravelStateStore();
 
-  const handlePause = () => {
+  const handlePause = async () => {
+    let { latitude, longitude } = (await Location.getCurrentPositionAsync({}))
```

ESLint

[error] 'latitude' is never reassigned. Use 'const' instead.

(prefer-const)